src/5e-SRD-Monsters.json

```diff
@@ -35748,7 +35748,7 @@
               }
             },
             {
-              "name": "Pass without Trace",
+              "name": "Pass Without Trace",
```

Comment:
Looks like this is a repeat? Dryad might just have their spells entirely wrong.

Reply:
This block is for Unicorn. Verified against the SRD, looks correct to me.